src/main/java/io/hydrox/inventorysummary/InventorySummaryPlugin.java

```diff
@@ -102,14 +115,28 @@ private void groupItems()
 		groupItems(container == null ? new Item[0] : container.getItems());
 	}
 
+	private boolean isWhitelisted(Item item)
+	{
+		if (config.whitelistEnabled()) {
+			ItemComposition itemComposition = itemManager.getItemComposition(item.getId());
+			return whitelist.contains(itemComposition.getName());
+		}
+
+		// If the whitelist isn't enabled, all items are allowed
+		return true;
+	}
+
 	private void groupItems(Item[] items)
 	{
+
 		Map<Integer, Integer> groupedItems = Arrays.stream(items)
 			.filter(p -> p.getId() != -1)
+			.filter(this::isWhitelisted)
 			.collect(groupingBy(Item::getId, LinkedHashMap::new, summingInt(Item::getQuantity)));
 
 		int spacesUsed = (int) Arrays.stream(items)
 			.filter(p -> p.getId() != -1)
+			.filter(this::isWhitelisted)
```

Comment:
I don't think spaces left should be whitelisted, given that it's pretty important to see when you're full, even if you don't care about the random stuff you're not trying to see.

Reply:
Agreed. That was a bit of an oversight.